RecoMuon/TrackerSeedGenerator/plugins/TSGForOIFromL2.cc

```diff
@@ -151,6 +143,7 @@ void TSGForOIFromL2::produce(edm::StreamID sid, edm::Event& iEvent, const edm::E
 
     // Check if the two positions (using updated and not-updated TSOS) agree withing certain extent.
     // If both TSOSs agree, use only the one at vertex, as it uses more information. If they do not agree, search for seeds based on both.
+    double L2muonEta=l2->eta();
```

Comment:
Thank you for the updates. Here, however, it was the ```std::abs()``` the operation that I suggested to avoid repeting:
```
    double absL2muonEta=std::abs(l2->eta());
```
Could you please also adjust this one? Then I will sign this as soon as automatic tests finish

Reply:
@perrotta 
done.